### ISSUE TITLE
Flutter iOS build fails

### ISSUE THREAD

**Issue by ctippur** [flutter/flutter#41033]
Issue: iOS build started to fail on Xcode 10.3.
I started to get "Multiple commands produce Flutter.framework" error. I followed the instructions to remove the framework  https://github.com/flutter/flutter/issues/20685

But I seem to run into a package .h file not found error - flutter_keychain.h in this case.

This issue seem to have been referenced in the above link as well. 

Appreciate any pointers. I am in a time crunch.

## Steps to Reproduce
https://github.com/flutter/flutter/issues/20685

```
$ flutter doctor -v
[✓] Flutter (Channel stable, v1.9.1+hotfix.2, on Mac OS X 10.14.6 18G84, locale en-US)
    • Flutter version 1.9.1+hotfix.2 at /Users/shekartippur/playground/cyclopscloud/cyclopsglobal/flutter/flutter
    • Framework revision 2d2a1ffec9 (2 weeks ago), 2019-09-06 18:39:49 -0700
    • Engine revision b863200c37
    • Dart version 2.5.0

 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/shekartippur/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 10.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.3, Build version 10G8
    • CocoaPods version 1.6.1

[✓] Android Studio (version 3.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 31.3.1
    • Dart plugin version 181.5656
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[✓] IntelliJ IDEA Community Edition (version 2017.3)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 29.0.1
    • Dart plugin version 173.3727.108

[✓] Connected device (1 available)
    • iPhone Xʀ • 17E002DE-29B0-4299-8257-375F28C36925 • ios • com.apple.CoreSimulator.SimRuntime.iOS-12-4 (simulator)

• No issues found!

<!-- If possible, paste the output of running `flutter doctor -v` here. -->

No errors. I ran a flutter clean as well.

```

**Comment by ctippur** [comment]
Any updates please?
I am completely blocked on my release.

**Comment by ctippur** [comment]
<img width="255" alt="Screen Shot 2019-09-22 at 6 58 00 AM" src="https://user-images.githubusercontent.com/2225047/65389022-6543df80-dd06-11e9-9b1e-9ad919ffb08d.png">

**Comment by ctippur** [comment]
This is what I see in the Xcode log:

/opt/ui/ios/Runner/GeneratedPluginRegistrant.m:6:9: fatal error: 'flutter_keychain/FlutterKeychainPlugin.h' file not found
#import <flutter_keychain/FlutterKeychainPlugin.h>
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.

**Comment by ctippur** [comment]
One more observation ... emulator works well.

**Comment by christopherfujino** [comment]
@jmagman could you take a first look at this one please?

**Comment by christopherfujino** [comment]
@ctippur could you provide full error logs?

**Comment by ctippur** [comment]
Sure .. I will add the logs by today evening.

**Comment by ctippur** [comment]
@christopherfujino - logs you are referring to is fro Xcode right?

**Comment by christopherfujino** [comment]
Any logs you have would be helpful. In particular, you've posted several snippets from logs (e.g. about CocoaPods or the missing header file) but the surrounding context would be helpful.

**Comment by jmagman** [comment]
Does the header actually exist in your Xcode project? Are you running from the xcworkspace?

**Comment by ctippur** [comment]
Weird .. I am getting a different error. This is whats happened. I have auto upgrade for xcode. Overnight, xcode upgraded to version 11. I did not realize it. I think this is when I started to face the error.
I realized it and downgraded to 10.3. I persist to see the issue.
I am now unable to recreate earlier error but I am getting a linking error:

Ld /opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/Runner.build/Release-iphoneos/Runner.build/Objects-normal/armv7/Runner normal armv7
    cd /Users/opt/Downloads/gpertui/ios
    export IPHONEOS_DEPLOYMENT_TARGET=8.0
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -arch armv7 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS12.4.sdk -L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos -L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/FMDB -L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/XCDYouTubeKit -L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/flutter_keychain -L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/flutter_local_notifications -L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/flutter_secure_storage -L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/flutter_youtube -L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/google_maps_flutter -L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/path_provider -L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/screen -L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/shared_preferences -L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/sqflite -L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/url_launcher -L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/video_player -L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/webview_flutter -L/Users/opt/Downloads/gpertui/ios/Flutter -F/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos -F/Users/opt/Downloads/gpertui/ios/Pods/../.symlinks/flutter/ios-release -F/Users/opt/Downloads/gpertui/ios/Pods/GoogleMaps/Base/Frameworks -F/Users/opt/Downloads/gpertui/ios/Pods/GoogleMaps/Maps/Frameworks -F/Users/opt/Downloads/gpertui/ios/Flutter -filelist /Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/Runner.build/Release-iphoneos/Runner.build/Objects-normal/armv7/Runner.LinkFileList -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -Xlinker -rpath -Xlinker @executable_path/Frameworks -miphoneos-version-min=8.0 -dead_strip -Xlinker -object_path_lto -Xlinker /Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/Runner.build/Release-iphoneos/Runner.build/Objects-normal/armv7/Runner_lto.o -Xlinker -final_output -Xlinker /Applications/Runner.app/Runner -fobjc-arc -fobjc-link-runtime -ObjC -lFMDB -lXCDYouTubeKit -lc++ -lflutter_keychain -lflutter_local_notifications -lflutter_secure_storage -lflutter_youtube -lgoogle_maps_flutter -lpath_provider -lscreen -lshared_preferences -lsqflite -lsqlite3 -lurl_launcher -lvideo_player -lwebview_flutter -lz -framework Accelerate -framework CoreData -framework CoreGraphics -framework CoreImage -framework CoreLocation -framework CoreTelephony -framework CoreText -framework Flutter -framework GLKit -framework GoogleMaps -framework GoogleMapsBase -framework GoogleMapsCore -framework ImageIO -framework JavaScriptCore -framework MediaPlayer -framework OpenGLES -framework QuartzCore -framework SystemConfiguration -framework UIKit -ObjC -lFMDB -lXCDYouTubeKit -lc++ -lflutter_keychain -lflutter_local_notifications -lflutter_secure_storage -lflutter_youtube -lgoogle_maps_flutter -lpath_provider -lscreen -lshared_preferences -lsqflite -lsqlite3 -lurl_launcher -lvideo_player -lwebview_flutter -lz -framework Accelerate -framework CoreData -framework CoreGraphics -framework CoreImage -framework CoreLocation -framework CoreTelephony -framework CoreText -framework Flutter -framework GLKit -framework GoogleMaps -framework GoogleMapsBase -framework GoogleMapsCore -framework ImageIO -framework JavaScriptCore -framework MediaPlayer -framework OpenGLES -framework QuartzCore -framework SystemConfiguration -framework UIKit -lETLDLFDynamic -framework Flutter -framework App -lPods-Runner -Xlinker -dependency_info -Xlinker /Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/Runner.build/Release-iphoneos/Runner.build/Objects-normal/armv7/Runner_dependency_info.dat -o /Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/Runner.build/Release-iphoneos/Runner.build/Objects-normal/armv7/Runner

ld: warning: directory not found for option '-L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/FMDB'
ld: warning: directory not found for option '-L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/XCDYouTubeKit'
ld: warning: directory not found for option '-L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/flutter_keychain'
ld: warning: directory not found for option '-L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/flutter_local_notifications'
ld: warning: directory not found for option '-L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/flutter_secure_storage'
ld: warning: directory not found for option '-L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/flutter_youtube'
ld: warning: directory not found for option '-L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/google_maps_flutter'
ld: warning: directory not found for option '-L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/path_provider'
ld: warning: directory not found for option '-L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/screen'
ld: warning: directory not found for option '-L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/shared_preferences'
ld: warning: directory not found for option '-L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/sqflite'
ld: warning: directory not found for option '-L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/url_launcher'
ld: warning: directory not found for option '-L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/video_player'
ld: warning: directory not found for option '-L/Users/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/webview_flutter'
ld: library not found for -lFMDB
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by ctippur** [comment]
Also, I am able to get the emulator. I am getting this error only during archive within xcode.

**Comment by ctippur** [comment]
Here is the log:

CompileC /opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/Runner.build/Release-iphoneos/Runner.build/Objects-normal/armv7/GeneratedPluginRegistrant.o Runner/GeneratedPluginRegistrant.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
    cd /opt/Downloads/gpertui/ios
    export LANG=en_US.US-ASCII
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x objective-c -arch armv7 -fmessage-length=0 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -std=gnu99 -fobjc-arc -fmodules -gmodules -fmodules-cache-path=/opt/Library/Developer/Xcode/DerivedData/ModuleCache.noindex -fmodules-prune-interval=86400 -fmodules-prune-after=345600 -fbuild-session-file=/opt/Library/Developer/Xcode/DerivedData/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wnon-modular-include-in-framework-module -Werror=non-modular-include-in-framework-module -Wno-trigraphs -fpascal-strings -Os -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Werror=return-type -Wunreachable-code -Wno-implicit-atomic-properties -Werror=deprecated-objc-isa-usage -Wno-objc-interface-ivars -Werror=objc-root-class -Wno-arc-repeated-use-of-weak -Wduplicate-method-match -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wconditional-uninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wnon-literal-null-conversion -Wobjc-literal-conversion -Wshorten-64-to-32 -Wpointer-sign -Wno-newline-eof -Wno-selector -Wno-strict-selector-match -Wundeclared-selector -Wno-deprecated-implementations -DCOCOAPODS=1 -DCOCOAPODS=1 -DNS_BLOCK_ASSERTIONS=1 -DOBJC_OLD_DISPATCH_PROTOTYPES=0 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS12.4.sdk -fstrict-aliasing -Wprotocol -Wdeprecated-declarations -miphoneos-version-min=8.0 -g -fvisibility=hidden -Wno-sign-conversion -Winfinite-recursion -Wcomma -Wblock-capture-autoreleasing -Wstrict-prototypes -Wno-semicolon-before-method-body -iquote /opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/Runner.build/Release-iphoneos/Runner.build/Runner-generated-files.hmap -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/Runner.build/Release-iphoneos/Runner.build/Runner-own-target-headers.hmap -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/Runner.build/Release-iphoneos/Runner.build/Runner-all-target-headers.hmap -iquote /opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/Runner.build/Release-iphoneos/Runner.build/Runner-project-headers.hmap -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/include -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/FMDB/FMDB.framework/Headers -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/XCDYouTubeKit/XCDYouTubeKit.framework/Headers -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/flutter_keychain/flutter_keychain.framework/Headers -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/flutter_local_notifications/flutter_local_notifications.framework/Headers -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/flutter_secure_storage/flutter_secure_storage.framework/Headers -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/flutter_youtube/flutter_youtube.framework/Headers -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/google_maps_flutter/google_maps_flutter.framework/Headers -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/path_provider/path_provider.framework/Headers -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/screen/screen.framework/Headers -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/shared_preferences/shared_preferences.framework/Headers -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/sqflite/sqflite.framework/Headers -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/url_launcher/url_launcher.framework/Headers -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/video_player/video_player.framework/Headers -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/webview_flutter/webview_flutter.framework/Headers -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/Runner.build/Release-iphoneos/Runner.build/DerivedSources-normal/armv7 -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/Runner.build/Release-iphoneos/Runner.build/DerivedSources/armv7 -I/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/Runner.build/Release-iphoneos/Runner.build/DerivedSources -F/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos -F/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/FMDB -F/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/XCDYouTubeKit -F/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/flutter_keychain -F/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/flutter_local_notifications -F/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/flutter_secure_storage -F/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/flutter_youtube -F/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/google_maps_flutter -F/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/path_provider -F/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/screen -F/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/shared_preferences -F/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/sqflite -F/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/url_launcher -F/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/video_player -F/opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/webview_flutter -F/opt/Downloads/gpertui/ios/Pods/../.symlinks/flutter/ios-release -F/opt/Downloads/gpertui/ios/Pods/GoogleMaps/Base/Frameworks -F/opt/Downloads/gpertui/ios/Pods/GoogleMaps/Maps/Frameworks -F/opt/Downloads/gpertui/ios/Flutter -MMD -MT dependencies -MF /opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/Runner.build/Release-iphoneos/Runner.build/Objects-normal/armv7/GeneratedPluginRegistrant.d --serialize-diagnostics /opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/Runner.build/Release-iphoneos/Runner.build/Objects-normal/armv7/GeneratedPluginRegistrant.dia -c /opt/Downloads/gpertui/ios/Runner/GeneratedPluginRegistrant.m -o /opt/Library/Developer/Xcode/DerivedData/Runner-ejiyxpghyedfcccjjjmmiqdijsey/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/Runner.build/Release-iphoneos/Runner.build/Objects-normal/armv7/GeneratedPluginRegistrant.o

/opt/Downloads/gpertui/ios/Runner/GeneratedPluginRegistrant.m:6:9: fatal error: 'flutter_keychain/FlutterKeychainPlugin.h' file not found
#import <flutter_keychain/FlutterKeychainPlugin.h>
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.

**Comment by ctippur** [comment]
@jmagman I dont see the header file.
Also, I dont see xcworkspace in xcode.

**Comment by jmagman** [comment]
Can you confirm you are working out of Runner.xcworkspace and NOT Runner.xcodeproj?
See instructions at https://flutter.dev/docs/get-started/install/macos#deploy-to-ios-devices.

**Comment by ctippur** [comment]
Apologies. Thats was it. It worked after I opened ios/Runner.xcworkspace from Flutter app. I think I got derailed because of the upgrade.

Thanks for pointing it out.
Closing the issue.

**Comment by jmagman** [comment]
> Apologies. Thats was it. It worked after I opened ios/Runner.xcworkspace from Flutter app. I think I got derailed because of the upgrade.
> 
> Thanks for pointing it out.

You definitely aren't alone, it's a common issue https://github.com/flutter/flutter/issues/10770

**Comment by BadReese** [comment]
Yes, I use Visual Code and click right button to "Open In Xcode", and forgot it is not open xcworkspace file, waste my half day time to figure it out ...

**Comment by tadadat** [comment]
> Runner.xcworkspace

I run open Runner.xcworkspace too, still problem persist after that, do I need to do anything after opening xcworkspace?

**Comment by brunovsiqueira** [comment]
@tadadat same here. Any updates? Please reopen it.

**Comment by jmagman** [comment]
If you are opening Runner.xcworkspace and getting an error then you aren't hitting the same issue as the issue author.  Please open a new issue.

**Comment by tadadat** [comment]
Thanks, and apologies for late update opening Runner.xcworkspace + changing platform :ios '9.0' in  PodFile fixes mine

**Comment by srjmalla** [comment]
This fixed the issue for me

flutter channel stable

flutter upgrade

**Comment by adsazad** [comment]
Archive button is not being enabled in Runner.xcworkspace.
please help

**Comment by jmagman** [comment]
@adsazad That can happen if you have a simulator targeted in Xcode.  Make sure `Any iOS device (arm64, armv7)` is selected as your target in Xcode.  
<img width="1194" alt="Screen Shot 2021-02-25 at 10 32 17 AM" src="https://user-images.githubusercontent.com/682784/109199834-c621e380-7754-11eb-8a4b-f98a7afcf1cc.png">

**Comment by adsazad** [comment]
It worked for me when I closed xcode and then did flutter build and open xcode workspace.

**Comment by JosteveGit** [comment]
This worked for me.

`flutter clean`
`flutter pub get`
`cd ios`
`pod install`
